<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PaintWindow</class>
 <widget class="QMainWindow" name="PaintWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>436</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Paint</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0">
    <property name="verticalSpacing">
     <number>0</number>
    </property>
    <item row="6" column="0">
     <widget class="QLabel" name="brushLabel">
      <property name="styleSheet">
       <string notr="true">margin-top:10px</string>
      </property>
      <property name="text">
       <string>Brush size:</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QSlider" name="brushSlider">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="value">
       <number>10</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="10" column="0">
     <widget class="QSlider" name="rSlider">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="12" column="0">
     <widget class="QSlider" name="gSlider">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="1" rowspan="21">
     <widget class="PaintWidget" name="graphicsView"/>
    </item>
    <item row="13" column="0">
     <widget class="QLabel" name="label_3">
      <property name="styleSheet">
       <string notr="true">margin-top:10px</string>
      </property>
      <property name="text">
       <string>Blue:</string>
      </property>
     </widget>
    </item>
    <item row="14" column="0">
     <widget class="QSlider" name="bSlider">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="20" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="0">
     <widget class="QRadioButton" name="circleButton">
      <property name="styleSheet">
       <string notr="true">margin: 5px 0</string>
      </property>
      <property name="text">
       <string>Circle</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">toolsGroup</string>
      </attribute>
     </widget>
    </item>
    <item row="15" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="11" column="0">
     <widget class="QLabel" name="label_2">
      <property name="styleSheet">
       <string notr="true">margin-top: 10px</string>
      </property>
      <property name="text">
       <string>Green:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QRadioButton" name="triangleButton">
      <property name="styleSheet">
       <string notr="true">margin: 5px 0</string>
      </property>
      <property name="text">
       <string>Triangle</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">toolsGroup</string>
      </attribute>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true">margin-top: 20px</string>
      </property>
      <property name="text">
       <string>Red:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QRadioButton" name="rectButton">
      <property name="styleSheet">
       <string notr="true">margin: 5px 0</string>
      </property>
      <property name="text">
       <string>Rectangle</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">toolsGroup</string>
      </attribute>
     </widget>
    </item>
    <item row="19" column="0">
     <widget class="QWidget" name="colorSwatch" native="true">
      <property name="minimumSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: pink;</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QRadioButton" name="brushButton">
      <property name="styleSheet">
       <string notr="true">margin: 5px 0</string>
      </property>
      <property name="text">
       <string>Brush</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">toolsGroup</string>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>620</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSave"/>
    <addaction name="actionClear"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="text">
    <string>Clear</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Backspace</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PaintWidget</class>
   <extends>QGraphicsView</extends>
   <header>/Users/yaroslav/Developer/Study/POOP/Paint/src/PaintWidget/PaintWidget.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="toolsGroup"/>
 </buttongroups>
</ui>
